ocs_ci/ocs/ui/page_objects/object_bucket_claims_tab.py

```diff
@@ -87,6 +89,10 @@ def check_obc_option(self, text="Object Bucket Claims"):
         namespaces.append(namespace_obj)
         delete_projects(namespaces)
 
+        if not obc_found:
+            return False
```

Comment:
usually when such boolean check needed, it is commonly used
`return obc_found` 

not critical, up to you.